sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,25 +117,42 @@ should be used.
 To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
-To execute SYCL tests on Intel devices following runtimes should be installed:
+The following `GPU` runtimes should be installed to run SYCL applications on  Intel `GPU` devices:
 
-**Linux**
-
-* Intel GPU [Intel&reg; Graphics Compute Runtime for
+* Linux: [Intel&reg; Graphics Compute Runtime for
    OpenCL&trade;](https://github.com/intel/compute-runtime/releases)
-* Intel CPU - [Experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications
-   with SYCL support](https://github.com/intel/llvm/releases)
-* Intel FPGA - TBD.
 
-**Windows**
-* OpenCL&trade; runtime for Intel GPU [Intel&reg; Download
+* Windows: [Intel&reg; Download
    Center](https://downloadcenter.intel.com/product/80939/Graphics-Drivers)
 
-* The experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications with
-  SYCL support on Windows will be available soon.
 
-Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel
-runtime versions.
+To install the OpenCL runtime for Intel `CPU` and `FPGA` devices the corresponding runtime asset/archive should be downloaded from [SYCL Compiler and Runtime updates](../ReleaseNotes.md) and installed using the following procedure.
```

Comment:
Not sure the instruction below applicable for OpenCL runtime for FPGA device.

Reply:
Well, If you are not sure, then I remove FPGA.